### ISSUE TITLE
When you type anything the request is made

### ISSUE THREAD

**Issue by m4v15** [FACN1/week4-FACNhosers#5]
Maybe it should only make request when we type letters

**Comment by edgarfinn** [comment]
Yep, agree. We limited ours so that it only adds when you type a letter or backspace. Only thing is now we also needs to clear when the input field goes back to empty

This might be helpful
http://www.asciitable.com/
